pkgs/development/python-modules/git-filter-repo/default.nix

```
@@ -18,6 +22,13 @@ buildPythonPackage rec {
     hash = "sha256-/hdT4Y8L1tPJtXhoyAEa59BWpuurcGcGOWoV71MScl4=";
   };
 
+  docs = fetchFromGitHub {
+    owner = "newren";
+    repo = pname;
```

```suggestion
    repo = "git-filter-repo";
```
We should not use `repo = pname` pattern. This is because `repo` and `pname` is semantically different.
see also https://github.com/NixOS/nixpkgs/issues/277994